language/resource.language.pt_br/strings.po

```diff
@@ -2602,3 +2602,13 @@ msgstr "Em Exibição"
 msgctxt "#31545"
 msgid "Premiering"
 msgstr "Estreia"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31546"
+msgid "TBC"
+msgstr "A Ser Confirmado"
```

Comment:
Is there a smaller / more compact translation for this string? This will likely end up too long for where it is used in the ratings line for tvshow status.

Reply:
Maybe it's a case of leaving it in the original. :)